GameData/000_FilterExtensions Configs/SubCategories_BDA.cfg

```diff
@@ -0,0 +1,82 @@
+SUBCATEGORY
+{
+	name = Guidance and Targeting
+	icon = targetingicon
+	FILTER
+	{
+		CHECK
+		{
+			type = moduleName
+			value = ModuleTargetingCamera,BDModulePilotAI,ModuleWingCommander,CMDropper
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = Missiles and Bombs
+	icon = missile
+	FILTER
+	{
+		CHECK
+		{
+			type = moduleName
+			value = MissileLauncher,RocketLauncher 
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = Turrets and Guns
+	icon = turret
+	FILTER
+	{
+		CHECK
+		{
+			type = moduleName
+			value = ModuleTurret,ModuleWeapon
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = Radar
+	icon = DataTransmitter
+	FILTER
+	{
+		CHECK
+		{
+			type = moduleName
+			value = ModuleRadar 
+		}
+	}
+
+SUBCATEGORY
+{
+	name = Ammo
+	icon = BahaSP
+	FILTER
+	{
+		CHECK
+		{
+			type = moduleName
+			value = CFEnable
+		}
+	}	
+}
+	
+SUBCATEGORY
+{
+	name = Other Parts
```

Comment:
There was this module "BDACategoryModule" I surmised it was something Baha added to catch parts that didn't match up, added it just in case